src/test/run-make/cabi-int-widening/cfoo.c

```diff
@@ -0,0 +1,4 @@
+// ignore-license
+int foo(char x) {
```

Comment:
`char` is a distinct type from `signed char` and `unsigned char` and its signedness is implementation defined. And according to e.g. http://stackoverflow.com/questions/3728045/any-compiler-which-takes-char-as-unsigned  `char` is in fact unsigned on ARM.